src/Map.elm

```diff
@@ -55,6 +60,14 @@ type alias Amount =
     Int
 
 
+type alias Height =
+    Float
```

Comment:
I would consider using a custom type for these instead of using a type alias. A type alias is helpful for expressing intent in your system, but does not provide any additional type safety.

If you find you have units that should never be accidentally transposed, its helpful to wrap them in a custom sum type.

For example:
```
type Height = Height Float
```

Reply:
Thank you, I think this would be a better approach